### DISCUSSION TITLE
State Management RFC

### DISCUSSION THREAD

**Discussion by rametta** [rametta/pratica#18] [RFCs]
## Summary

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris et urna in odio efficitur feugiat quis vitae massa. Integer aliquam venenatis scelerisque. Vestibulum id mi mi. Sed at dolor quis mauris egestas malesuada. Vestibulum dapibus et quam a venenatis.

## Need

Integer fermentum tempus elementum. Sed et iaculis nisl. Nulla facilisi. Nunc mattis semper ullamcorper. Fusce placerat sollicitudin elit at iaculis. Vestibulum vehicula vulputate ligula vel venenatis. Donec gravida arcu non erat ultrices, in congue tellus tempus. Interdum et malesuad

Nulla tempus blandit erat id sollicitudin. Etiam aliquam purus eget tortor hendrerit tempor. Nullam faucibus quam dui, quis tempor diam dignissim eget. Suspendisse non neque diam. Phasellus fermentum lacinia nibh nec semper. Fusce sed lacinia metus. Suspendisse id dui urna. 

-  **Suspendisse** non neque diam. Phasellus fermentum lacinia nibh nec semper. Fusce sed lacinia metus. Suspendisse id dui urna. 

### Phase 1 

Integer fermentum tempus elementum. Sed et iaculis nisl. Nulla facilisi. Nunc mattis semper ullamcorper. Fusce placerat sollicitudin elit at iaculis. Vestibulum vehicula vulputate ligula vel venenatis. Donec gravida arcu non erat ultrices, in congue tellus tempus.

### Phase 2 

Integer fermentum tempus elementum.

**Comment by rametta** [comment]
Wow great idea!

**Reply by joonarahko** [reply]
Test

**Comment by joonarahko** [comment]
Testing this

**Reply by rametta** [reply]
ascasc

**Reply by joonarahko** [reply]
wow

**Comment by joonarahko** [comment]
> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris et urna in odio efficitur feugiat quis vitae massa. Integer aliquam venenatis scelerisque. Vestibulum id mi mi. Sed at dolor quis mauris egestas malesuada. Vestibulum dapibus et quam a venenatis. Nam aliquam risus non odio dapibus, non aliquam nibh semper. Sed gravida nulla in accumsan ullamcorper. Suspendisse placerat, nisl pellentesque volutpat cursus, mauris orci varius ipsum, vel maximus dui tellus a odio. Nulla facilisi. Nam aliquet ultricies felis, a tincidunt tortor. Quisque accumsan egestas quam ut varius. Phasellus gravida aliquam suscipit. Etiam a nulla sed arcu dapibus ultrices. Quisque tristique sed odio eu lacinia. Praesent bibendum purus et nibh finibus tempus.
